web/static/css/app.css

```diff
@@ -389,3 +393,21 @@ code {
   margin-bottom: 0.3rem;
 }
 
+@font-face {
+  font-family: 'fontello';
+  src: url('/fonts/tick-fonts/fontello.eot');
```

Comment:
this implies we are loading the _entire_ font library just for the `tick` is that correct?